x/auth/client/cli/tx_sign.go

```diff
@@ -271,7 +269,7 @@ func multisigSign(clientCtx client.Context, txBuilder client.TxBuilder, txFactor
 		txFactory,
 		clientCtx,
 		multisigAddr,
-		clientCtx.GetFromName(),
+		clientCtx.FromName,
```

Comment:
why this change?  This seems unnecessary

Reply:
It was from the backport. It doesn't change anything yeah

Reply:
I think in the original PR we removed those getters as they are basically useless, but i reverted that here anyway.